docs/serving/istio-authorization.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Knative application under the strict authorization policy"
+weight: 25
+type: "docs"
+---
+
+When you deployed app to Knative Serving, serving system pods such as activator and autoscaler access to your app.
```

Comment:
```suggestion
Knative Serving system pods, such as the activator and autoscaler components, require access to your deployed Knative services.
```

Reply:
Why do these components need access to Knative services / apps? Can we explain this, because otherwise users may think it's optional.